igcollect/postgres.py

```diff
@@ -125,12 +125,13 @@ def main():
                 print(template.format('bgwriter', key, value))
 
         # table size
-        for line in execute(conn, (
-                'SELECT relname,'
-                '    pg_total_relation_size(relid)'
-                "FROM pg_catalog.pg_statio_user_tables ORDER BY pg_total_relation_size(relid)"
-                "DESC;"
-        )):
+        for line in execute(conn, ('''
+                SELECT relname, pg_total_relation_size(relid)
```

Comment:
This query does not work for me. There is no such column `relid` in `pg_class` table nor in `pg_namespace` table - am I holding it wrong ?

Reply:
Already fixed...Refresh